core/iwasm/common/arch/invokeNative_ia32.s

```diff
@@ -16,9 +16,12 @@ _invokeNative:
     push    %ebp
     movl    %esp, %ebp
     movl    16(%ebp), %ecx          /* ecx = argc */
-    movl    12(%ebp), %edx          /* edx = argv */
+    leal    2(%ecx), %edx           /* edx = ecx + 2 (count return address and saved ebp) */
+    andl    $3, %edx                /* edx = edx % 4 */
+    leal    -16(%esp, %edx, 4), %esp /* esp = esp - 16 + edx * 4 */
```

Comment:
If edx is 0 (argc == 2/6/..), no need to sub 16, so as to save stack, suggest to improve it like below:
```asm
    ...
    andl   $3, %edx
    jz   stack_aligned
   leal     -16(%esp, %edx, 4), %esp
stack_aligned:
   test  %ecx, %ecx
   ...
```